databases/postgresql96/Portfile

```diff
@@ -7,12 +7,11 @@ PortGroup deprecated 1.0
 
 # Final release was on 2021-11-08
 deprecated.upstream_support no
-known_fail          yes
 
 #remember to update the -doc and -server as well
 name                postgresql96
 version             9.6.24
-revision            1
+revision            2
```

Comment:
No need to bump the revision as this is just a build fix.

Reply:
Done thanks.